platform-configs/msi-pro-z690-a-ddr5.robot

```diff
@@ -13,3 +13,18 @@ ${DEVICE_AUDIO2}=                   Raptorlake HDMI
 
 ${CPU_MAX_FREQUENCY}=               5200
 ${CPU_MIN_FREQUENCY}=               300
+
+
+${CPU_TESTS_SUPPORT}=                           ${TRUE}
+${INTEL_HYBRID_ARCH_SUPPORT}=                   ${TRUE}
+${HYPER_THREADING_SUPPORT}=                     ${TRUE}
```

Comment:
These 3 flags might go to msi-common? Only the CPU count number are per-unit specific?

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/628944c..0f55d7ec3cb33c76e147a229949b4eb966f24b71